resource-managers/yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClientSchedulerBackend.scala

```diff
@@ -164,7 +164,7 @@ private[spark] class YarnClientSchedulerBackend(
 
     super.stop()
     client.stop()
-    logInfo("Stopped")
+    logInfo("YarnClientSchedulerBackend Stopped")
```

Comment:
@amanomer, why don't you just make the log better just by not using class name directly like the reference you pointed out? e.g.) `YARN client scheduler backend stopped`.

Reply:
@jobitmathew please update the log message.

Reply:
@amanomer 
@HyukjinKwon 
@srowen 
@tgravescs 
changes done as per HyukjinKwon's suggestion.Could you please check.